l1-contracts/contracts/upgrades/L1GenesisUpgrade.sol

```diff
@@ -101,4 +111,12 @@ contract L1GenesisUpgrade is IL1GenesisUpgrade, BaseZkSyncUpgradeGenesis {
         super.upgrade(_proposedUpgrade);
         return Diamond.DIAMOND_INIT_SUCCESS_RETURN_VALUE;
     }
+
+    function _getAdditionalForceDeploymentsData() internal view returns (bytes memory) {
+        AdditionalForceDeploymentsData memory additionalForceDeploymentsData = AdditionalForceDeploymentsData({
+            baseTokenAssetId: s.baseTokenAssetId,
+            l2Weth: address(0) // kl todo
```

Comment:
please create a task so that we don't forget

Reply:
https://linear.app/matterlabs/issue/EVM-742/weth-address-for-l2assetrouter-ntv